packages/tradershub/src/helpers/accountHelpers.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * `getAccountTitle` is a helper function which accepts name of the account and is_virtual (isDemo) flag and return a combined string by appending the title 'Demo' if the flag is on.
+ * @param accountName the name of the account such as: marketTypeDetails.title or platform.ctrader.title etc.
+ * @param isDemo the flag to check if the account is a demo account.
+ * @returns a string, the combined string of account name and 'Demo' if the flag is on.
+ */
+export const getAccountTitle = (accountName: string, isDemo = false) => {
```

Comment:
[nit]: if the function is only for the cfds then rename to cfdsAccountHelpers file and getCfdsAccountTitle function name.

Reply:
@matin-deriv I named the file this way so that if later we want to add anything else to this helper about the accounts we do it in the same file, for the function yes sure I will rename it, for the file what do you think should I change or keep it this way?

Reply:
yes please rename so it won't be confusing with the Deriv accounts.